### ISSUE TITLE
Integrate Discourse forum into website

### ISSUE THREAD

**Issue by sarahetoms** [simplworld/simpl.world.website#12]
I'm not sure if skinning is required, so cc-ing @chadwhitman on ticket
We will need a Forum section added to simpl.world which points to discourse.

Ticket dependent on #11 being completed

**Comment by jefftriplett** [comment]
I'm calling this one on hold.

**Comment by joelee484** [comment]
Done!
